static/js/data_handler.js

```diff
@@ -74,6 +75,46 @@ export let dataHandler = {
     },
     createNewCard: function (cardTitle, boardId, statusId, callback) {
         // creates new card, saves it and calls the callback function with its data
-    }
+    },
     // here comes more features
+    submitNewTitle: function submit_new_title(event) {
+                        let header = event.currentTarget.parentElement;
+                        let newTitle = header.querySelector(".new-board-title");
+                        console.log(newTitle.value);
+                        if (newTitle.value === ''){
+                            newTitle.value = newTitle.getAttribute("value");
+                        }
+                        let saveButton = header.querySelector(".title-save-button");
+                        header.removeChild(newTitle);
+                        header.removeChild(saveButton);
+                        let boardTitle = document.createElement('span');
+                        boardTitle.classList.add('board-title');
+                        boardTitle.textContent = newTitle.value;
+                        header.prepend(boardTitle);
+                        let unslicedBoardId = header.parentElement.id;
+                        let boardId = unslicedBoardId.replace("board-","");
+
+                        let titleAndId = {
+                            newTitle: newTitle.value,
+                            boardId: boardId
+                        };
+                        dom.renameBoard();
+                        fetch(`${window.origin}/change-board-title`, {
+                            method: "PUT",
+                            credentials: "include",
+                            body: JSON.stringify((titleAndId)),
+                            cache: "no-cache",
+                            headers: new Headers({
+                                "content-type": "application/json"
+                            })
+                        }).then(function (response) {
+                            if (response.status !== 200){
```

Comment:
This logs are useful during debugging, but they should be deleted after we don't need them anymore.